src/pages/industries/security.jsx

```diff
@@ -0,0 +1,184 @@
+import { TetragonCard } from 'components/pages/industries/cards/tetragon';
+import FeaturedTalks from 'components/pages/industries/featured-talks';
+import Hero from 'components/pages/industries/hero';
+import IndustryUseCases from 'components/pages/industries/industry-usecase/industry-usecase';
+import Community from 'components/shared/community';
+import Container from 'components/shared/container/container';
+import SecurityImage2 from 'images/pages/industries/security-2.png';
+import SecurityImage3 from 'images/pages/industries/security-3.png';
+import SecurityImage4 from 'images/pages/industries/security-4.png';
+import SecurityBee from 'images/pages/industries/security-bee.png';
+import TetragonImage from 'images/pages/industries/tetragon.png';
+import MainLayout from 'layouts/main/main';
+
+import EncryptionIcon from './images/encryption.inline.svg';
+import EnforcementIcon from './images/enforcement.inline.svg';
+import GrayedEgdelessLogo from './images/logos/edgeless.svg';
+import GrayedF5Logo from './images/logos/f5.svg';
+import GrayedFRSCALogo from './images/logos/frsca.svg';
+import GrayedGdataLogo from './images/logos/gdata.svg';
+import GrayedKryptosLogo from './images/logos/kryptos.svg';
+import GrayedPalantirLogo from './images/logos/palantir.svg';
+import GrayedProtonLogo from './images/logos/proton.svg';
+import GrayedTestifyLogo from './images/logos/testify.svg';
+import NetworkPolicyIcon from './images/network-policy.inline.svg';
+import ProcessIcon from './images/process.inline.svg';
+import RuntimeEnforcementIcon from './images/runtime-enforcement.inline.svg';
+import RuntimeIcon from './images/runtime.inline.svg';
+import tetragonImage1 from './images/tetragon-illustration.png';
+
+const heroContent = {
+  heading: 'Security',
+  texts: [
+    'For businesses at the forefront of cybersecurity and those for whom security is integral to their operations, the rise of cloud native applications presents immense opportunities and accompanying challenges. The dynamism of cloud native architectures, from microservices to containerization and distributed computing, demands a paradigm shift in securing these environments.',
+    'Traditional security approaches struggle to adapt to the ephemeral and dynamic nature of cloud native applications. ',
+    'Cilium delivers robust cloud native security with features like zero trust security, transparent encryption, mutual TLS, security observability, and runtime protection. Leveraging eBPF, Cilium offers efficient observability across the entire application stack, integrates seamlessly with SIEM systems, and ensures compliance with standards like FIPS, FedRAMP, and SOC. Cilium aligns security practices with the realities of building, maintaining, and scaling cloud native applications.',
+  ],
+  imageSrc: SecurityBee,
+  imageAlt: 'ebeedex security bee',
+};
+
+const tetragonContent = {
+  heading: 'Tetragon: eBPF-based Security Observability and Runtime Enforcement',
```

Comment:
I would replace this Tetragon section with ones on network policy and encryption as those are the top two security features